e2e/verify/verify.go

```diff
@@ -289,6 +289,66 @@ func (c *ctx) singularityVerifySigner(t *testing.T) {
 	}
 }
 
+func (c *ctx) checkGroupidOption(t *testing.T) {
+	/*
+		// Create a temp directory where we can store a valid image
+		tempDir, err := ioutil.TempDir("", "")
+		if err != nil {
+			t.Fatalf("failed to create a temporary directory: %s", err)
+		}
+		defer func() {
+			err := os.RemoveAll(tempDir)
+			if err != nil {
+				t.Fatalf("failed to delete temporary directory: %s", err)
+			}
+		}()
+
+		imgPath := filepath.Join(tempDir, "imgTest.sif")
+
+		// Pull a valid image
+		e2e.PullImage(t, c.env, successURL, imgPath)
+	*/
```

Comment:
Why is this commented out?

Reply:
Also looks like this should be removed in conjunction with whatever is done wrt my review comment